backend/user_management/minio/routers/upload.py

```
@@ -1,41 +1,125 @@
 router = APIRouter()
 
-@router.post("/upload/photos/{product_id}", response_model=ProductPhotoUploadResponse)
-async def upload_photos(
-    product_id: int, 
-    files: List[UploadFile] = File(...), 
-    db: Session = Depends(get_db)
+
+@router.post("/upload", response_model=ProductPhotoUploadResponse)
+async def unified_upload(
+    upload_type: str = Form(...),
+    files: List[UploadFile] = File(...),
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)